ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_initiator_group.py

```diff
@@ -229,6 +234,11 @@ def main():
                         ],
             "type": "str"
         },
+        "change_name": {
+            "required": False,
+            "type": "str",
+            "no_log": False
+        },
```

Comment:
interesting... is this the typical way objects are renamed in ansible?

Reply:
yes.
we had to introduce a new variable "change_name" because "name" is used to find which object to work on.

ex: for a given prot._schedule , update description to "modified desc.". for this to achieve a user needs the below ansible task

name: change name of Volume
hpe_nimble_volume:
hostname: "{{ hostname }}"
username: "{{ username }}"
password: "{{ password }}"
state: "{{ state | default('present') }}"
description: "modified description"
name: "{{ name }}" ============> name is used by our code to find the object set for which we need to update the description. 

In any update operation we need two inputs. One is name and the other one is the attribute we want to modify/update. But if we want to modify/update the name variable itself . Then we cannot pass name variable twice. Hence, we need to introduce a new variable to accept the new name. existing variable "name" is again used to find out the object for which we need to change the name.

Thanks,
Alok

Reply:
Got it.  Thanks for the detailed explanation.